crypto/rand/rand_unix.c

```diff
@@ -244,6 +257,7 @@ static ssize_t sysctl_random(char *buf, size_t buflen)
 }
 #  endif
 
+#  if defined(OPENSSL_RAND_SEED_GETRANDOM)
```

Comment:
The `#ifdef OPENSSL_RAND_SEED_XXX` and `#if defined(OPENSSL_RAND_SEED_XXX)` conditions were mixed at random. I chose the latter for all.